### PR TITLE
Turtle definition for the Croissant core vocabulary

### PR DESCRIPTION
**Pull request by benjelloun**

### CODE REVIEW

File: docs/croissant.ttl [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+# Top-level classes
+
+croissant:FileObject a rdf:Class ;
+  rdfs:label "FileObject" ;
+  rdfs:comment "An individual file that is part of a dataset." ;
+  rdfs:subClassOf schema:CreativeWork .
+
+croissant:FileSet a rdf:Class ;
+  rdfs:label "FileSet" ;
+  rdfs:comment "A set of homogeneous files extracted from a container, optionally filtered by inclusion and/or exclusion filters. " ;
```
Comment (marcenacp) [inline docs/croissant.ttl:15]: Space at the end of sentence

File: docs/croissant.ttl [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+# Secondary classes
+
+croissant:DataType a rdf:Class ;
+  rdfs:label "DataType" ;
+  rdfs:comment "The data type of values expected for a Field in a RecordSet.  This class is inspired by the Datatype class in CSVW. In addition to simple atomic types, types can be semantic types, such as schema.org classes, as well types defined in other vocabularies." ;
+  rdfs:subClassOf schema:URL .
+
+croissant:Format a rdf:Class ;
+  rdfs:label "Format" ;
+  rdfs:comment "Specifies how to parse the format of the data froma string representation. For example, format may hold a date format string, a number format, or a bounding box format." ;
```
Comment (marcenacp) [inline docs/croissant.ttl:32]: froma -> from a

File: docs/croissant.ttl [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+croissant:references a rdf:Property ;
+  rdfs:label "references" ;
+  rdfs:comment "Another Field of another RecordSet that this field references. This is the equivalent of a foreign key reference in a relational database." ;
+  schema:domainIncludes croissant:Field ;
+  schema:rangeIncludes croissant:Field .
+
+croissant:subField a rdf:Property ;
+  rdfs:label "subField" ;
+  rdfs:comment "Another Field that is nested inside this one." ;
+  schema:domainIncludes croissant:Field ;
```
Comment (marcenacp) [inline docs/croissant.ttl:156]: croissant:RecordSet? Because subFields are in RecordSet

**Review by marcenacp — APPROVED** [review]
Thanks!

File: docs/croissant.ttl (hunk 2 above)
Comment (benjelloun) [reply docs/croissant.ttl:32]: Done

File: docs/croissant.ttl (hunk 1 above)
Comment (benjelloun) [reply docs/croissant.ttl:15]: Done

Comment (benjelloun) [reply docs/croissant.ttl]: Done

**Pull request merged**